tests/serve/test_vllm.py

```diff
@@ -294,10 +288,56 @@ def vllm_config_test(request):
 @pytest.mark.vllm
 @pytest.mark.e2e
 def test_serve_deployment(
-    vllm_config_test, request, runtime_services, predownload_models
+    vllm_config_test, request, runtime_services, predownload_models, image_server
 ):
     """
     Test dynamo serve deployments with different graph configurations.
     """
     config = vllm_config_test
     run_serve_deployment(config, request)
+
+
+@pytest.mark.vllm
+@pytest.mark.e2e
+@pytest.mark.gpu_2
+def test_multimodal_b64(request, runtime_services, predownload_models):
+    """
+    Test multimodal inference with base64 url passthrough.
+
+    This test is separate because it loads the required image at runtime
+    (not collection time), ensuring it only fails when actually executed.
```

Comment:
It's fine to merge as-is, but one other thought for future -- if you move `vllm_configs` definition from global to inside `vllm_config_test` instead, I think you could avoid needing a whole separate test_function for this edge case as well.